migrator.sh

```diff
@@ -200,7 +202,7 @@ docker_login() {
 
 # decode username/password for a registry to query the API
 decode_auth() {
-  AUTH_CREDS="$(cat ~/.dockercfg | jq -r '."'${1}'".auth' | base64 --decode)"
+  AUTH_CREDS="$(cat ~/.docker/config.json | jq -r '.auths."'${1}'".auth' | base64 --decode)"
```

Comment:
This won't work with the docker client 1.6.2 which is in use currently.  This is being used by design for compatibility reasons with the Docker CS Engine.  Here is where the build failed:
```
WARNING: login credentials saved in /root/.dockercfg.
Login Succeeded
cat: /root/.docker/config.json: No such file or directory
WARNING: login credentials saved in /root/.dockercfg.
Login Succeeded
```